ui/app/mainui/AppMain.qml

```diff
@@ -761,6 +766,20 @@ Item {
             }
         }
 
+        Component {
+            id: activityCenterPopupComponent
+            ActivityCenterPopup {
+                id: activityCenter
+                height: appView.height - 56 * 2 // TODO get screen size // Taken from old code top bar height was fixed there to 56
+                y: 56
```

Comment:
this will be problematic when resizing the window, can't we use margins?

Reply:
@alexandraB99 I actually don't know, I can try :)

Reply:
@alexandraB99 unfortunately, we can't cos `Popup` doesn't have `anchors.topMargin` or `anchors.bottomMargin`.
I tested it it works fine.